pronto_mia/lib/ui/views/deployment_plan/edit/deployment_plan_edit_view.dart

```diff
@@ -10,15 +14,22 @@ import 'package:pronto_mia/ui/views/deployment_plan/edit/deployment_plan_edit_vi
 
 @FormView(fields: [
   FormTextField(name: 'description'),
-  FormTextField(name: 'pdfPath'),
   FormTextField(name: 'availableFrom'),
   FormTextField(name: 'availableUntil'),
+  FormTextField(name: 'pdfPath'),
 ])
 class DeploymentPlanEditView extends StatelessWidget
     with $DeploymentPlanEditView {
   final DeploymentPlan deploymentPlan;
 
-  DeploymentPlanEditView({Key key, this.deploymentPlan}) : super(key: key);
+  DeploymentPlanEditView({Key key, this.deploymentPlan}) : super(key: key) {
+    if (deploymentPlan != null) {
+      descriptionController.text = deploymentPlan.description;
+      availableFromController.text = deploymentPlan.availableFrom.toString();
+      availableUntilController.text = deploymentPlan.availableUntil.toString();
+      pdfPathController.text = 'upload.pdf';

```

Comment:
If we want to implement the edit functionality this way the user should nonetheless have a possibility to view the currently uploaded file. Unfortunately in edit mode the user cannot click on the filename like he is used to from creation.

Reply:
This was not intended. A implented a fix.